packages/material-ui-lab/src/TreeNode/TreeNode.js

```diff
@@ -0,0 +1,327 @@
+import React from 'react';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import warning from 'warning';
+import Typography from '@material-ui/core/Typography';
+import Collapse from '@material-ui/core/Collapse';
+import { withStyles } from '@material-ui/core/styles';
+import { useForkRef } from '@material-ui/core/utils';
+import TreeViewContext from '../TreeView/TreeViewContext';
+
+const styles = {
```

Comment:
To generate the CSS api

```suggestion
export const styles = {
```